.bash_profile

```diff
@@ -23,3 +19,8 @@ alias grep="/usr/local/opt/grep/bin/ggrep"
 alias logs="jq . service.log | less"
 alias dcb="docker-compose build"
 alias dcu="docker-compose up -d"
+alias dps="docker ps"
+
+# load nvm
```

Comment:
I'd recommend adding a comment here about why this block needs to be the last lines of the terminal setup file so the context in the MR description aren't lost